app/scripts/services/mobileClients.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("MobileClientsService", function(DataService) {
+
+    var mobileclientVersion = {
```

Comment:
this can be `const` instead of `var`

Reply:
Id only change to `const` if it is being used elsewhere in the code base. I'm unsure if they are using those features